### ISSUE TITLE
Add "Send to VirusTotal" command

### ISSUE THREAD

**Issue by robinhood2014** [microsoft/winfile#256]
One command I like seeing in such Sysinternals programs as Process Explorer and Autoruns is the "Send to VirusTotal" command. I would like to suggest adding the same command to File Manager in a future release.

**Comment by SscSPs** [comment]
Can you elaborate on the feature?
Is it similar to "Send to" option in windows explorer's right click menu? or is it something different?
I don't think your message contains enough info to create a feature(request) unless I'm missing something obvious.

**Comment by robinhood2014** [comment]
Here's a screenshot of what I'm talking about.
![checkvirustotal1](https://user-images.githubusercontent.com/6971466/84600563-bcaf5000-ae3f-11ea-92c7-16ca5adcad02.png)

**Comment by UltraHR** [comment]
I don't think it makes very much sense for a piece of software like winfile to prescribe what virus scanning service you use.

**Comment by craigwi** [comment]
If there is some generic way to enumerate such addins (like for the regular Explorer), we could support for that.  This was mentioned in #260.

**Comment by bapcyk** [comment]
what is "virus total"? MS Defender?

**Comment by NTDLS** [comment]
WinFile should only scan with MWAV for Win 3.1.
